src/simdutf/flb_simdutf_connector.cpp

```diff
@@ -61,23 +62,23 @@ int flb_simdutf_connector_convert_utf16le_to_utf8(const char16_t *buf, size_t le
                                                   char **utf8_output, size_t *out_size)
 {
     size_t clen = 0;
-    size_t converted = 0;
-    simdutf::result result;
+    simdutf::result result = {};
 
     clen = simdutf::utf8_length_from_utf16le(buf, len);
-    /* convert_utfXXXX_to_utf8 function needs to pass allocated memory region with C++ style */
-    std::unique_ptr<char[]> output{new char[clen]};
-    converted = simdutf::convert_utf16le_to_utf8(buf, len, output.get());
-    result = simdutf::validate_utf8_with_errors(output.get(), clen);
-    if (result.error == simdutf::error_code::SUCCESS && converted > 0) {
-        std::string result_string(output.get(), clen);
+    *utf8_output = (char*)flb_malloc(clen + 1);
```

Comment:
Please check the [coding style guide](https://github.com/fluent/fluent-bit/blob/master/CONTRIBUTING.md) for this and other issues.

In this case the missing spaces in the data type type and after the closing parenthesis of the cast.